src/components/password.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+export default class Password extends React.Component {
+  state = {
+    input: ''
+  };
+
+  onKeyPress = event => {
+    const target = event.target;
+    const valid = target.checkValidity();
+
+    this.setState({ input: event.target.value, submit: valid });
+  };
+
+  onSubmit = () => {
+    this.props.handleSubmit(this.state.input);
+  };
+
+  render() {
+    return (
+      <div className="password">
+        <label htmlFor="password">
+          {this.props.question}
+          <input
+            value={this.state.value}
```

Comment:
Suggest to lowercase it all
And it would be cool if we could check how much of the password matches e.g. 90% is good enough